plugins/auth-backend/src/lib/passport/PassportStrategyHelper.ts

```diff
@@ -91,6 +91,7 @@ export const executeRedirectStrategy = async (
 export const executeFrameHandlerStrategy = async <Result, PrivateInfo = never>(
   req: express.Request,
   providerStrategy: passport.Strategy,
+  options?: Record<string, string>,
```

Comment:
could this be

```suggestion
  options?: Record<string, unknown>,
```

? Unless strategy.authenticate explicitly only allows for strings

Reply:
again this matches the implementation in okta